src/bios_handler.cpp

```diff
@@ -101,11 +101,10 @@ void IbmBiosHandler::biosAttributesCallback(sdbusplus::message_t& i_msg)
             if (auto l_val = std::get_if<std::string>(
                     &(std::get<5>(std::get<1>(l_attribute)))))
```

Comment:
this is bit confusing for me..
in line 97, when you do get<0>(l_property.second) -> which implies that you are getting the 0th index from std::variant (types::BiosBaseTable). which is nothing but std::map<std::string, BiosProperty>; right?

which means that l_attributeList is std::map..
then how come line 102 - has std::get<1>(l_attribute); std::map doesn't have std::get api.

can you please help me here to understand this piece..

Reply:
okay 👍